### DISCUSSION TITLE
Data - all missing

### DISCUSSION THREAD

**Discussion by fuxe13** [tbnobody/OpenDTU#2089] [Q&A]
### What happened?

Hallo Zusammen!
Habe seid heute das Problem, dass keine Daten empfangen werden.  Bis gestern war alles ok, kein Update und auch sonst keine Veränderungen....mir ist dann aufgefallen, dass der nRF24 Chip nicht installiert ist...hab eine neue Platine aufgesetzt. 
Neustarts, Update auf 24.6.10 hat alles nichts gebracht.

Möchte sicher nicht auf die Software den Fehler schieben, aber was und wie kann ich noch testen?
Hoffe jemand kann mir helfen :)
LG Walter

### To Reproduce Bug

neustart

### Expected Behavior

neustart

### Install Method

Pre-Compiled binary from GitHub

### What git-hash/version of OpenDTU?

v4.4.7-dirty

### Relevant log/trace output

```Shell
12:56:43.156 > Request SystemConfigPara
12:56:43.217 > TX RealTimeRunData Channel: 23 --> 15 80 11 74 95 80 11 89 44 80 0B 00 66 77 FF 6A 00 00 00 00 00 00 00 00 A6 7A EA 
12:56:43.668 > RX Period End
12:56:43.668 > All missing
12:56:43.668 > Nothing received, resend whole request
12:56:43.668 > TX RealTimeRunData Channel: 40 --> 15 80 11 74 95 80 11 89 44 80 0B 00 66 77 FF 6A 00 00 00 00 00 00 00 00 A6 7A EA 
12:56:44.218 > RX Period End
12:56:44.218 > All missing
12:56:44.218 > Nothing received, resend whole request
12:56:44.218 > TX RealTimeRunData Channel: 61 --> 15 80 11 74 95 80 11 89 44 80 0B 00 66 77 FF 6A 00 00 00 00 00 00 00 00 A6 7A EA 
12:56:44.770 > RX Period End
12:56:44.770 > All missing
12:56:44.770 > Nothing received, resend whole request
12:56:44.770 > TX RealTimeRunData Channel: 75 --> 15 80 11 74 95 80 11 89 44 80 0B 00 66 77 FF 6A 00 00 00 00 00 00 00 00 A6 7A EA 
12:56:45.321 > RX Period End
12:56:45.321 > All missing
12:56:45.321 > Nothing received, resend whole request
12:56:45.321 > TX RealTimeRunData Channel: 3 --> 15 80 11 74 95 80 11 89 44 80 0B 00 66 77 FF 6A 00 00 00 00 00 00 00 00 A6 7A EA 
12:56:45.835 > RX Period End
12:56:45.835 > All missing
```

### Please confirm the following

- [X] I believe this issue is a bug that affects all users of OpenDTU, not something specific to my installation.
- [X] I have already searched for relevant existing issues and discussions before opening this report.
- [X] I have updated the title field above with a concise description.

**Comment by tbnobody** [comment]
Aufgrund der Tatsache, dass dieser Fehler nicht alle Benutzer betrifft sondern deine Installation schiebe ich das Ganze mal in die Discussions.

**Comment by tbnobody** [comment]
Es kann auch an instabilen Netzteilen liegen. (Auch ein 2A Netzteil kann instabile/wellige Spannung ausgeben). Es könnte an fehlenden Kondensatoren o.ä. liegen welche die Spannung glätten. Das Funkmodul könnte defekt sein (ist schon oft genug aufgetreten)
Ich persönlich hatte auch schon das Problem, dass ein Gerät in der Nachbarschaft amok gelaufen ist und die Frequenzen blockiert hat. 

Und btw. deine Versionsangabe bezieht sich auf die SDK, nicht auf die OpenDTU Version.

**Comment by fuxe13** [comment]
Das Funkmodul hab ich getauscht,
Netzteil testweise auch,
Nachbarn sind kein Problem, die sind alle weit weg,
und die Version der Software ist V 24.6.10

da ja die Hardware scheinbar funktioniert, die Software auch und es keine Änderungen gab, versteh ich das ganze nicht...

**Comment by fuxe13** [comment]
Nachtrag, seid gut einer Stunde geht es wieder ohne was getan zu haben.....merkwürdig....
